### PR TITLE
Implement XRP send

### LINKED ISSUE #720: Eth fees in Sat/b
Ethereum send form has fees as Sats/b

Screenshot from 2019-10-25 10-01-36

### PR DESCRIPTION
**Pull request by vladimirvolek**
closes: https://github.com/trezor/trezor-suite/issues/720

### CODE REVIEW

File: packages/suite/src/actions/wallet/sendFormSpecific/bitcoinActions.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Output } from '@wallet-types/sendForm';
 import { networkAmountToSatoshi } from '@wallet-utils/accountUtils';
 import { getLocalCurrency } from '@wallet-utils/settingsUtils';
 import { SEND } from '@wallet-actions/constants';
-import { INPUT_SEQUENCE } from '@wallet-constants/sendForm';
+import { BITCOIN_INPUT_SEQUENCE } from '@wallet-constants/sendForm';
```
Comment (szymonlesisz) [inline packages/suite/src/actions/wallet/sendFormSpecific/bitcoinActions.ts]: all others constants prefix are "BTC", is there any reason why this one has "BITCOIN" ?

Comment (szymonlesisz) [reply packages/suite/src/actions/wallet/sendFormSpecific/bitcoinActions.ts]: maybe it should be called "BTC_RBF_SEQUENCE" ?

Comment (vladimirvolek) [reply packages/suite/src/actions/wallet/sendFormSpecific/bitcoinActions.ts]: no :)

File: packages/suite/src/actions/wallet/sendFormSpecific/rippleActions.ts [hunk 2]
```diff
@@ -1,12 +1,74 @@
...
+
+    const output = getOutput(send.outputs, 0);
+    const amountInSatoshi = networkAmountToSatoshi(
+        output.amount.value || '0',
+        account.symbol,
+    ).toString();
+    const { availableBalance } = account;
+    const feeInSatoshi = send.selectedFee.value;
+    let tx;
+    const totalSpentBig = new Bignumber(calculateTotal(amountInSatoshi || '0', feeInSatoshi));
```
Comment (szymonlesisz) [inline packages/suite/src/actions/wallet/sendFormSpecific/rippleActions.ts]: amountInSatoshi shouldn't be undefined, you have already declare fallback value in line 25 `output.amount.value || '0'` so there is no need to do the same, right?

**Pull request merged**